src/config/ci-iceberg-test.toml

```diff
@@ -1,6 +1,6 @@
 [meta]
 disable_recovery = true
-max_heartbeat_interval_secs = 600
```

Comment:
Not sure whether it can work. Let's run the test before mergeing.

Reply:
All tests passed. Let me merge it.